docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+services:
+  app:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    image: summit-report-api
+    command: python manage.py runserver 0.0.0.0:8000
```

Comment:
change the port to something diff than api and idp

Reply:
done. New port is 8003.